server/question-repository/routes/question.ts

```diff
@@ -0,0 +1,104 @@
+import express, { Request, Response } from "express";
+import db from "../db/conn";
```

Comment:
I think if you use the require('../db/conn') method this will bring the reference of the db already instantiated by the db init file (conn.ts). 

Subsequently you wouldnt have to await the db in each of the endpoints.